tests/unit/client_tests.py

```diff
@@ -163,6 +163,7 @@ def test_multiple_connect(self):
             self.client.disconnect()
             self.assertIsNone(self.client.r_session)
 
+    @skip_for_iam
     def test_auto_renew_enabled(self):
```

Comment:
It looks like we are skipping all methods in this class for `iam` testing.  What if we skip the whole class instead?

Reply:
Disregard the comment above, we are not skipping all tests in the class.